lib/pypiCoordinatesMapper.js

```diff
@@ -16,16 +16,15 @@ class PypiCoordinatesMapper {
   }
 
   _shouldResolve(coordinates) {
-    return coordinates.name.includes('.') ||
-      coordinates.name.includes('_') ||
-      coordinates.name.includes('-')
+    if (typeof coordinates.name !== 'string' || coordinates.name.includes('/')) return false
+    return coordinates.name.includes('.') || coordinates.name.includes('_') || coordinates.name.includes('-')
```

Comment:
How would you like something like this?

```suggestion
    if (typeof coordinates.name !== 'string' || coordinates.name.includes('/')) {
        return false;
    }

    const validSeparatorCharacters = ['.', '_', '-'];
    return validSeparatorCharacters.some(char => coordinates.name.includes(char));
}
```

Reply:
Tried using `equivalentSeparators.some(char => coordinates.name.includes(char))`, but the code scanner does not like it.  Will leave this as is.